extensions/tasks/NLUTestV0/publishTestResults.ts

```diff
@@ -0,0 +1,50 @@
+import * as tl from 'azure-pipelines-task-lib/task';
```

Comment:
Do we need to put these in a separate file? It's only ~50 lines so wouldn't be a big deal to add to index.ts.